articles/azure/azs-how-configure-custom-script-extension.md

```diff
@@ -58,7 +58,7 @@ See documentation here on [how to create a new VM on Azure Stack Hub](azs-how-cr
 
 ## [Existing VM](#tab/tabid-2)
 
-***
```

Comment:
Need to have the `***` here so that the tabs function correctly. Try serving it locally and you should see what I mean.

Also the new sentence needs a full stop at the end please.

Reply:
Done